packages/testdata/src/gethGenesis/invalidSpuriousDragonGethGenesis.ts

```diff
@@ -32,4 +32,4 @@ export const invalidSpuriousDragonGethGenesis: GethGenesis = {
   gasUsed: '0x0',
   parentHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
   baseFeePerGas: null,
-} as GethGenesis
+} as unknown as GethGenesis
```

Comment:
Why the `unkown` here?

Reply:
This object is (intentionally) missing many properties or has incorrect ones, and I think the last commit that updated string to PrefixedHexString push it over the TS edge where it now requires as unknown. I think this is fine because in this context, we are intentionally trying to pass an incorrect gethGenesis "as a real gethGenesis".

Reply:
Ah right, good point, thanks for clarification :smile: :+1: